### PR TITLE
JAMES-2995 Fix ElasticSearch mapping request when behind a proxy 

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: backends-common/elasticsearch/src/test/resources/auth-es/nginx-conf/reverse_elasticsearch.conf [hunk 1]
```diff
@@ -8,5 +8,6 @@ server {
 
     location / {
         proxy_pass http://elasticsearch:9200;
+        proxy_cache_convert_head off;
```
Comment (rouazana) [inline backends-common/elasticsearch/src/test/resources/auth-es/nginx-conf/reverse_elasticsearch.conf]: in fact this does not change anything

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by chibenwa** [comment]
test this please (double scheduling for more safety)

**Comment by rouazana** [comment]
merged

**Pull request closed without merging**